### DISCUSSION TITLE
Should modellers know the true biomass time-series from Atlantis?

### DISCUSSION THREAD

**Discussion by grantdadams** [NOAA-EDAB/ms-keyrun#88] [Ideas] [answered]
Thinking on yesterday's meeting and the original theme behind the project, I am wondering if estimation modellers should have knowledge of the "true" time series of biomass? 

I would propose no, because in the real world we don't know how many fish there actually are and modellers true to best represent the system given the data and prior knowledge they have. I would suggest that, if possible, we limit presentation of the true biomass time-series. Once the EMs are up and running, we submit the final derived quantities to an Atlantis person for calculation of performance metrics.

**Comment by vbartolino** [comment, marked as answer]
@grantdadams I agree. Of course this doesn't prevent EM teams to interact and compare each other results :)
